scripts/gitlab_trigger_client_publish

```diff
@@ -2,10 +2,22 @@
 
 set -ex
 
-mender_version=$1
+if [[ $# -eq 2 ]]; then
+  version_to_publish=$1
+  trigger_from_repo=$2
+elif [[ $# -eq 1 ]]; then
+  # Backwards compatibility
+  version_to_publish=$1
+  trigger_from_repo="mender"
+else
+  echo "Usage: $0 version repo" >&2
+  exit 1
+fi
 
-if [ -z "$mender_version" ]; then
-  echo "Usage: $0 mender-version"
+if [ "$trigger_from_repo" != "mender" -a \
+     "$trigger_from_repo" != "mender-artifact" -a \
+     "$trigger_from_repo" != "mender-connect" ]; then
+  echo "Unrecognized repository $trigger_from_repo" >&2
```

Comment:
Suggestion:

```bash
if ! grep "^${trigger_from_repo}\$" <<<$(release_tool.py -l) \
    && [ "$trigger_from_repo" != "meta-mender" ]; then
  echo "Unrecognized repository $trigger_from_repo" >&2
  ...
```

Reply:
Good point. This way it is more "future ready". Thanks!